src/main/resources/templates/db_scripts/schema.json

```diff
@@ -175,6 +180,21 @@
           "tOps" : "ADD"
         }
       ]
+    },
+    {
+      "tableName": "settings",
+      "fromModuleVersion": "mod-organizations-storage-4.7.0",
+      "withMetadata": true,
+      "fullTextIndex" : [
+        {
+          "fieldName" : "key",
+          "tOps" : "ADD"
+        },
+        {
+          "fieldName" : "value",
```

Comment:
I think we don't need index for value field, it will not be used

Reply:
also don't sure that we need fullTextIndex here and not just common btree index. 
Basically I don't think that we will have thousands of settings, it is ok to not use indexes here at all